src/Aspire.Hosting.Analyzers/Aspire.Hosting.Analyzers.csproj

```diff
@@ -0,0 +1,42 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <IsRoslynComponent>true</IsRoslynComponent>
+    <EnforceExtendedAnalyzerRules>true</EnforceExtendedAnalyzerRules>
+    <UsePublicApiAnalyzers>false</UsePublicApiAnalyzers>
+    <IsPackable>false</IsPackable>
+    <IsShipping>false</IsShipping>
```

Comment:
This feels like a lie. This is shipping. (I'm not sure what this property actually does.)

Reply:
I think it's used by build infra to organize outputs into the Shipping/NotShiping directories?

Reply:
Again, copied from dotnet/aspnetcore https://github.com/dotnet/aspnetcore/blob/843f7b011dae5687c77f3570542325b01301f8db/src/Framework/AspNetCoreAnalyzers/src/Analyzers/Microsoft.AspNetCore.App.Analyzers.csproj#L4-L7

Reply:
> copied from dotnet/aspnetcore

Your link points to `IsShippingPackage`, not `IsShipping`.

Reply:
Doh. I can remove. Do we know what it does?

Reply:
> Do we know what it does?

I don't. I see:

https://github.com/dotnet/aspnetcore/blob/843f7b011dae5687c77f3570542325b01301f8db/docs/ProjectProperties.md

not sure if those are dotnet/aspnetcore specific properties or what.